app.html

```diff
@@ -288,130 +285,116 @@ <h2 data-i18n>Settings</h2>
+            </div> 
           </section>
           <section class="links">
             <p>
-              Stay on top of your game with real-time position tracking, nodes
-              of weapons and points of interest, and overlay mode for seamless
-              progress tracking.<br />There are more apps on the games you love:
-            </p>
-            <a href="https://sotf.th.gl/privacy-policy.html" target="_blank">
-              Privacy Policy</a
-            >
-            <a href="https://palia.th.gl/" title="Palia Map" target="_blank">
-              Palia Map
-            </a>
-            <a
-              href="https://diablo4.th.gl/"
-              title="Diablo 4 Map"
-              target="_blank"
-            >
-              Diablo 4 Map
-            </a>
-            <a
-              href="https://aeternum-map.th.gl/"
-              title="Aeternum Map - New World Map"
-              target="_blank"
-            >
-              New World Interactive Map
-            </a>
-            <a
-              href="https://www.nwmap.info/"
-              title="New World Faction Territory Map"
-              target="_blank"
-            >
-              New World Territory Map
-            </a>
-            <a
-              href="https://www.hogwarts.th.gl/"
-              title="Hogwarts Legacy fansite"
-              target="_blank"
-            >
-              Hogwarts Legacy
-            </a>
-            <a
-              href="https://www.arkesia.th.gl/"
-              title="Interactive map for Lost Ark"
-              target="_blank"
-            >
-              Lost Ark
-            </a>
-            <a
-              href="https://lol.th.gl/"
-              title="Trophies app for League of Legends"
-              target="_blank"
-            >
-              League of Legends
-            </a>
-            <a
-              href="https://www.soc.th.gl/"
-              title="A Songs of Conquest fansite"
-              target="_blank"
-            >
-              Songs of Conquest
-            </a>
-            <a
-              href="https://github.com/lmachens/skeleton"
-              title="Simply display any website as customizable Overlay"
-              target="_blank"
-            >
-              Customizable Overlays
-            </a>
-            <a
-              href="https://www.th.gl/"
-              title="The Hidden Gaming Lair"
-              target="_blank"
-            >
-              The Hidden Gaming Lair
-            </a>
+              Take your gaming to new heights with accurate tracking, richly 
+              detailed waypoints, and a sleek overlay for real-time progress.<br /> 
+              Find more gaming enchancements at: <a href="https://www.th/gl/" target="_blank">The Hidden Gaming Lair</a>
+              Ensure your privacy by reviewing our: <a href="https://sotf.th.gl/privacy-policy.html" target="_blank">Privacy Policy</a>
```

Comment:
I understand that it might be a good idea to remove all the links and only show the link to th.gl, but all these links were added to get better SEO.
It might not be that important and I am open to accept your change. But the href should be fixed ;) )it's th.gl, not th/gl.